artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -3638,12 +3638,29 @@
             <xsd:element name="slow-consumer-threshold" type="xsd:long" maxOccurs="1" minOccurs="0">
                <xsd:annotation>
                   <xsd:documentation>
-                     The minimum rate of message consumption allowed before a consumer is considered "slow." Measured
-                     in messages-per-second.
+                     The minimum rate of message consumption allowed before a consumer is considered "slow."  Measurement
+                     unit is defined by the slow-consumer-threshold-measurement-unit parameter.  By default this is
+                     messages-per-seconds
                   </xsd:documentation>
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="slow-consumer-threshold-measurement-unit" maxOccurs="1" minOccurs="0">
+               <xsd:annotation>
+                  <xsd:documentation>
+                     The units used to measure the slow consumer threshold.  Default is messages-per-second.
+                  </xsd:documentation>
+               </xsd:annotation>
+               <xsd:simpleType>
+                  <xsd:restriction base="xsd:string">
```

Comment:
@clebertsuconic / @mtaylor this looks very similar to the period unit that is in the journal history, i assume we're just using TimeUnit under the covers, should we maybe extract the type and simply share it....

Reply:
e.g. could just have a shared type in the xsd that defines timeUnit and simple share that between Cleberts journal history time unit / period and yours.

    <xsd:simpleType name="timeUnit">
        <xsd:restriction base="xsd:string">
            <xsd:enumeration value="DAYS"/>
            <xsd:enumeration value="HOURS"/>
            <xsd:enumeration value="MINUTES"/>
            <xsd:enumeration value="SECONDS"/>
        </xsd:restriction>
    </xsd:simpleType>

Reply:
I did consider using a time unit.  However, the measurement unit is really messages-per-second, messages-per-minute, I wanted to make it clear hence went with the more verbose option.  

I suppose it could be changed to mean "messages-per-<TimeUnit>" but imo that's not as intuitive.  It doesn't use TimeUnit under the covers here it instead uses a "MeasurementUnit" enum (as the calculations were very simple).